code/client/munkilib/munkicommon.py

```diff
@@ -842,6 +843,15 @@ def getFirstPlist(textString):
     return (textString[plist_start_index:plist_end_index],
             textString[plist_end_index:])
 
+def getAppleLanguages():
+    user = getconsoleuser()
```

Comment:
getconsoleuser can return None or 'loginwindow' as well as an actual username. Does the next line handle these cases correctly?